src/Model/Actions.re

```diff
@@ -111,8 +111,8 @@ type t =
   | EnableZenMode
   | DisableZenMode
   | CopyActiveFilepathToClipboard
-  | SearchShow
-  | SearchHide
+  | PaneShow(Pane.paneType)
+  | PaneHide
```

Comment:
This seems like a good opportunity to make these non-function-like.

Reply:
Renamed to more event-like `PaneOpen(paneType)` and `PaneHidden` - does that sound better?

Reply:
This seems to describe the event that _will_ happen after it has been processed. Or the _intent_ perhaps, which is really just a different naming scheme for a function. What you want to describe is the event that _caused_ it, which requires a more fundamental change than just a rename. So in this case what we might (eventually) want is `ActivityBar(SearchButtonClicked)` and `Pane(SearchTabSelected | ProblemsTabSelected)`.

Reply:
These actions are removed now 👍